src/babel.js

```diff
@@ -63,7 +65,41 @@ export default function (babel) {
     name: 'emotion', // not required
     inherits: require('babel-plugin-syntax-jsx'),
     visitor: {
-      TaggedTemplateExpression (path) {
+      Program: {
+        enter (path, state) {
+          state.inline =
+            path.hub.file.opts.filename === 'unknown' || state.opts.inline
```

Comment:
Does this mean it will automatically switch to inline mode in situations where there is no file?